etc/config/csharp.amazon.properties

```diff
@@ -5,6 +5,7 @@ compilerType=csharp
 defaultCompiler=dotnet700csharp
 
 group.csharp.compilers=dotnettrunkcsharp:dotnet700csharp:dotnet6011csharp
+group.csharp.nugetPackages=/opt/compiler-explorer/dotnet-nuget
```

Comment:
I don;t see this directory on our network share:

```
Dec/26 22:51 admin-node~ $ ls -ld /opt/compiler-explorer/dotnet-*
lrwxrwxrwx  1 root root    21 Dec 26 07:35 /opt/compiler-explorer/dotnet-trunk -> dotnet-trunk-20221226
drwxr-xr-x 10 root root 22528 Dec 19 07:23 /opt/compiler-explorer/dotnet-trunk-20221219
drwxr-xr-x 10 root root 22528 Dec 20 07:31 /opt/compiler-explorer/dotnet-trunk-20221220
drwxr-xr-x 10 root root 22528 Dec 21 07:32 /opt/compiler-explorer/dotnet-trunk-20221221
drwxr-xr-x 10 root root 22528 Dec 22 07:31 /opt/compiler-explorer/dotnet-trunk-20221222
drwxr-xr-x 10 root root 22528 Dec 23 07:29 /opt/compiler-explorer/dotnet-trunk-20221223
drwxr-xr-x 10 root root 22528 Dec 25 07:37 /opt/compiler-explorer/dotnet-trunk-20221224
drwxr-xr-x 10 root root 22528 Dec 26 07:35 /opt/compiler-explorer/dotnet-trunk-20221226
drwxr-xr-x  3 root root  6144 Nov 25 07:21 /opt/compiler-explorer/dotnet-v6.0.1
drwxr-xr-x  9 root root 22528 Nov 25 23:00 /opt/compiler-explorer/dotnet-v6.0.11
drwxr-xr-x  9 root root 22528 Nov 25 22:59 /opt/compiler-explorer/dotnet-v7.0.0
```

What does this refer to?

Reply:
should probably be `/opt/compiler-explorer/dotnet_nuget`

Reply:
```
$ ls -l /opt/compiler-explorer/dotnet_nuget-v6.0.0/packages/
total 0
drwxrwxr-x 3 root root 28 Jan 31  2022 fsharp.core
drwxrwxr-x 3 root root 28 Jan 29  2022 microsoft.aspnetcore.app.runtime.linux-x64
drwxrwxr-x 3 root root 28 Jan 29  2022 microsoft.netcore.app.runtime.linux-x64

```

Reply:
> I don't understand the reference to the non-existent nugetPackages directory.

It's reserved for the future. Currently we don't have the directory existing. 
If we want to provide nuget packages support, we can create the directory from the infra in the future.

Reply:
Can we remove it please? There's no need to reserve anything for the future, we can make a change later to add it just as you've done here. Right now it's confusing and doesn't add anything.

Reply:
Done.

Reply:
Thanks; testing locally now.

Reply:
Worked! will merge. thanks for your patience.